controller/src/functions/audio-renderings-bus.ts

```diff
@@ -225,6 +241,7 @@ async function createContentRow({context, row}: createContentRowArgs) {
     resourceType: row.resourceType,
     createdOn: row.createdOn,
     modifiedOn: row.modifiedOn,
+    title: row.resourceTitle ? row.resourceTitle : null,
```

Comment:
Adds the title to new  audio projects if present (which doesn't currently exist for audio, but for consistency)